solr/solr-ref-guide/modules/deployment-guide/pages/docker-faq.adoc

```diff
@@ -140,7 +140,7 @@ docker run -it -v $PWD/mysolrhome:/mysolrhome -e SOLR_HOME=/mysolrhome solr
 
 This does need a pre-configured directory at that location.
 
-To make this easier, Solr docker supports a INIT_SOLR_HOME setting, which copies the contents
+Until version 7, Solr docker supported an INIT_SOLR_HOME setting, which copies the contents
```

Comment:
Better, just remove the entire mention of `INIT_SOLR_HOME`. We don't normally document earlier versions in ref guide